SLAM/GraphBasedSLAM/graphslam/edge/edge_odometry.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) 2020 Jeff Irion and contributors
+#
+# This file originated from the `graphslam` package:
+#
+#   https://github.com/JeffLIrion/python-graphslam
+
+r"""A class for odometry edges.
```

Comment:
Should it be raw string?

Reply:
You are using raw string in several docstrings. Is it for latex math?

Reply:
Yes, it was for LaTeX math, although as you found it isn't always needed.  I could change these to regular strings if you'd like.